### PR TITLE
Import Cluster fails on "node X doesn't have network details populated" with Ansible 2.8

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bugzilla: 1702412
tendrl-bug-id: Tendrl/commons#1084

Signed-off-by: GowthamShanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @nthomas-redhat This solution will work in current supporting ansible version as well as ansible 2.8.

File: tendrl/commons/utils/ansible_module_runner.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import os
 import subprocess
+import sys
 import tempfile
 
 import ansible.executor.module_common as module_common
 from ansible import modules
+from ansible.parsing.dataloader import DataLoader
+from ansible.template import Templar
```
Comment (shtripat) [inline tendrl/commons/utils/ansible_module_runner.py]: Should the import variable be called `Templar` or `Template` ?

Comment (GowthamShanmugam) [reply tendrl/commons/utils/ansible_module_runner.py]: it is templar only @shtripat https://github.com/ansible/ansible/blob/4d3a6123d5b84ed8cdaba4e945e6f1d55c35c4b4/lib/ansible/template/__init__.py#L267

Comment (shtripat) [reply tendrl/commons/utils/ansible_module_runner.py]: Ack.

**Pull request merged**